src/components/Button.spec.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import '@testing-library/jest-dom'
+
+import {render, fireEvent} from '@testing-library/svelte'
+
+import Button from "./Button.svelte";
+describe("Button component", () => {
+    window.alert = jest.fn();
+    window.alert.mockClear();
+
+    test("should confirm there is a button in the Button component ", () => {
+        const { container } = render(Button);
+        expect(container).toContainHTML("<button");
+        expect(container).toContainHTML("</button>");
+    });
+
+    test('show alert when the button gets clicked', async () => {
+        const { getByTestId, findByRole } = render(Button, {title: 'testButton'});
```

Comment:
> In the spirit of [the guiding principles](https://testing-library.com/docs/guiding-principles), it is recommended to use this only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible.

I would recommend to use `getByRole`  as the `button` element has the default role `button`